addons/purchase_stock/models/stock_rule.py

```diff
@@ -259,16 +259,17 @@ def _update_purchase_order_line(self, product_id, product_qty, product_uom, comp
             res['orderpoint_id'] = orderpoint_id.id
         return res
 
+    def _get_PO_date(self, company_id, values):
+        purchase_date = min([fields.Datetime.from_string(value['date_planned']) - relativedelta(days=int(value['supplier'].delay)) for value in values])
```

Comment:
I would add `today` in the list of value to:
1. fallback on it if `values` is empty
2. take this value instead of a date in the past like the issue we have here

-> you can remove the overide in `sale_purchase_stock`

Reply:
@Whenrow done